packages/server/package.json

```diff
@@ -81,7 +81,8 @@
     "mime": "2.4.4",
     "minimatch": "3.0.4",
     "minimist": "1.2.5",
-    "mocha": "2.4.5",
+    "mocha-2.4.5": "npm:mocha@2.4.5",
```

Comment:
yes - this was an oversight / mistake on our behalf when we bumped the driver **without** bumping the server. they should be in fact **always** be in sync. 

Ideally we would create a new shared mocha package to synchronize the versions. `packages/mocha`

FWIW we should likely dig into the problems with `7.0.1` because that's not something that should be happening - running different versions of the driver + server, yet synchronizing them through the event emitter is not good

Reply:
@flotwig or @Bkucera do we have a PR where we bump mocha to `7.x.x` so I can see what the errors / problems are?

Reply:
this is the error you'll get with mocha 4.1.0 and 7.1.0:

```
TypeError: Cannot read property 'passes' of undefined
    at Spec.Base.epilogue (/home/owner/dev/cypress/node_modules/mocha-7.1.0/lib/reporters/base.js:1:1)
    at Object.onceWrapper (events.js:1:1)
    at Runner.emit (events.js:1:1)
    at Reporter.emit (/home/owner/dev/cypress/packages/server/lib/reporter.coffee:214:21)
    at Object.onMocha (/home/owner/dev/cypress/packages/server/lib/project.js:1:1)
    at Socket.<anonymous> (/home/owner/dev/cypress/packages/server/lib/socket.js:1:1)
    at Socket.emit (events.js:1:1)
    at /home/owner/dev/cypress/packages/socket/node_modules/socket.io/lib/socket.js:1:1
    at processTicksAndRejections (internal/process/task_queues.js:1:1)
```

Reply:
pushed a commit that might fix 7.1.0

Reply:
@Bkucera i thought i heard you mention that you managed to reconcile these versions all to use 7.0.1, is that going to come in a separate PR, or can you clarify the situation now?